app/views/provider_interface/user_permissions/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for :browser_title, t('page_titles.provider.edit_user_permissions') %>
+<% content_for :before_content, govuk_back_link_to(@previous_page_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render ProviderInterface::ProviderUserPermissionsFormComponent.new(form_model: @wizard, form_path: provider_interface_organisation_settings_organisation_permissions_path(@provider, @provider_user), provider: @provider, user_name: @provider_user.full_name) %>
+    <p class="govuk-body">
+      <%= govuk_link_to t('cancel'), provider_interface_organisation_settings_organisation_permissions_path(@provider, @provider_user) %>
```

Comment:
should this be `provider_interface_organisation_settings_organisation_user_path` as this will error otherwise?